src/components/newEvent/CreateEvent.js

```diff
@@ -19,17 +19,11 @@ export function CreateEvent(props) {
                 
             />
             <label htmlFor="location">Location</label>
-            <select>
-                <option value="">--Please choose an option--</option>
-            </select>
-            //////Figure out the location/////
-            <label htmlFor="location">Location</label>
             <input
                 type="Location"
```

Comment:
maybe two inputs, one for city (type='text') and one for state (type='text' or a <select><option> tag for all 50 states)